packages/assets/scss/03-organisms/_rich-text.scss

```diff
@@ -60,11 +60,11 @@ $heading-indent: $gutter;
   & &__footnote {
     border-style: solid;
     border-width: 2px;
-    font-size: .6em;
+    font-size: .7em;
     line-height: 1;
-    padding: 0 .3em;
+    padding: 1.3px 6px;
```

Comment:
```suggestion
    padding: 1px 6px;
```

Reply:
I have no problem using the bigger size.  I was afraid that you might be concerned the visual presentation as making the size bigger.  The bigger font size helps low-vision users.